sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -3247,6 +3247,14 @@ def repeat(self, repeats):
       pd.core.strings.StringMethods, 'get_dummies',
       reason='non-deferred-columns')
 
+  split = frame_base.wont_implement_method(
```

Comment:
Does this commit mean that split/rsplit doesn't work in Beam, we learned about it and we declare that we don't intend to support it?